### PR TITLE
build(bubblebench-baseline): convert to ESM only

### PR DESCRIPTION
**Pull request by DLehenbauer**
Convert to ESM only

### CODE REVIEW

File: examples/benchmarks/bubblebench/common/tsconfig.cjs.json
Comment (tylerbutler) [inline examples/benchmarks/bubblebench/common/tsconfig.cjs.json:1]: Probably meant to be in a separate PR?

**Review by tylerbutler — CHANGES_REQUESTED** [review]
Just a couple of clarifications.

File: examples/benchmarks/bubblebench/baseline/package.json [hunk 1]
```diff
@@ -12,25 +12,10 @@
...
-				"default": "./lib/index.js"
-			},
-			"require": {
-				"types": "./dist/index.d.ts",
-				"default": "./dist/index.js"
-			}
-		}
-	},
-	"main": "dist/index.js",
-	"types": "dist/index.d.ts",
```
[the remark this answers is not in the document]
Comment (DLehenbauer) [reply examples/benchmarks/bubblebench/baseline/package.json:28]: It's intentional.  We've been removing all exports for applications (benchmarks, examples, etc.)

For these, the entry point is declared in 'webpack.config.json'.

**Review by tylerbutler — APPROVED** [review]
